src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -84,15 +116,69 @@ public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusCont
                 // this can happen if the host is offline
             }
 
-            var vote = GetAggregateScaleVote(votes, context, _logger);
+            ScaleVote vote = GetAggregateScaleVote(votes.Union(targetScaleVotes), context, _logger);
```

Comment:
Let's move this aggregation out of this method - this method should only be about monitors, and shouldn't be taking any targetScaleVotes as a parameter. This method should just return it's List<ScaleVote>.